auacm/requirements.txt

```diff
@@ -5,13 +5,14 @@ Flask-Bcrypt==0.7.1
 Flask-Login==0.3.0
 Flask-SocketIO==1.2
 Flask-SQLAlchemy==2.0
-eventlet==0.18.1
 itsdangerous==0.24
 Jinja2==2.8
+lazy-object-proxy==1.2.1
```

Comment:
I... I don't know where that came from. Probably Tornado, but I'll check it out.